src/index.js

```diff
@@ -300,18 +301,22 @@ function getMain({ options, entry, format }) {
 	mainsByFormat.es = replaceName(
 		pkg.module && !pkg.module.match(/src\//)
 			? pkg.module
-			: pkg['jsnext:main'] || 'x.esm.js',
+			: pkg['jsnext:main'] || pkgTypeModule
+			? 'x.esm.js'
+			: 'x.esm.mjs',
```

Comment:
I think `.m.js` and `.esm.js` were both poor approximations of `.mjs`, and we should be able to drop the infix.

```suggestion
			: 'x.mjs',
```

Reply:
Makes sense, though I wonder if that will confuse some users; for users who might not be super familiar with module formats, Microbundle is quite easy to understand given the current names. `esm` -> `x.esm.js`, `umd` -> `x.umd.js`, etc.

Reply:
It's a reasonable thought, yeah. I could go either way on this.